src/ParameterBag/NotifierParameterBagInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * This file is part of the Notifier package.
+ *
+ * (c) Dries De Peuter <dries@nousefreak.be>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Notifier\ParameterBag;
+
+interface NotifierParameterBagInterface {
+
+  /**
```

Comment:
Please use sf2 coding standards. 
Check out php-cs-fixer to simplify this process.